src/hooks/useSparkleFirestoreConnect.ts

```diff
@@ -1,6 +1,7 @@
+// eslint-disable-next-line no-restricted-imports
 import {
   ReduxFirestoreQuerySetting,
-  useFirestoreConnect,
+  useFirestoreConnect as _useFirestoreConnect,
```

Comment:
What is the thought process behind renaming this import here?

Assuming it was to match `_isLoaded` and `_isEmpty`, the intention of those being renamed was so that we could export our own versions of them within this same file's scope (though it looks like we didn't end up re-exporting `_isEmpty`.. which we should do)